CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 4.1.0
+
```

Comment:
We need to add `*To be released*` here, since as soon as this PR is merged, the file will be displayed here on github.
When 4.1.0 will be releease, we'll change it with the actual date